src/dev/flang/ast/Call.java

```diff
@@ -496,33 +496,34 @@ else if (_target != null)
   private void findChainedBooleans(Resolution res, AbstractFeature thiz)
   {
     var cb = chainedBoolTarget(res, thiz);
-    if (cb != null && _actuals.size() == 1)
+    if (cb != null && _actuals.size() == 1 && cb._actuals.getLast().typeIfKnown() != Types.t_ERROR)
```

Comment:
`cb._actuals.getLast().typeIfKnown() != Types.t_ERROR` the same is checked in the inner `if`. So this is redundant?

Reply:
I was playing around with a single `if` vs. two nested ones and finally decided for the nested `if`s and forgot to cleanup properly.